letsdrill.rb

```diff
@@ -1,24 +1,37 @@
 def get_letter_grade(integer)
- 
+
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
+  when 80..89
+    return "B"
+  when 70..79
+    return "C"
+  when 60..69
+    return "D"
+  else
+    return "F"
+  end
 
 end
 
 def shortest_string(array)
 
   #Put your code here!
+  array.min { |a, b|  a.size <=> b.size  }
```

Comment:
Thanks, min_by looks much cleaner. Did some research in the size/length
difference, they seem to be interchangeable, which I just learned. Thanks
again.

On Mon, Nov 16, 2015 at 12:56 PM, Michael Gee <notifications@github.com>
wrote:

> In letsdrill.rb
> <https://github.com/paircolumbus/LetsDrill/pull/42#discussion_r44957467>:
>
> >
> >  end
> >
> >  def shortest_string(array)
> >
> >    #Put your code here!
> > +  array.min { |a, b|  a.size <=> b.size  }
>
> Check out Enumerable#min_by
> <http://ruby-doc.org/core-2.0.0/Enumerable.html#method-i-min_by> for a
> way to shorten and make this slightly clearer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/LetsDrill/pull/42/files#r44957467>.
>